cve_bin_tool/extractor.py

```diff
@@ -1,7 +1,7 @@
 # Copyright (C) 2021 Intel Corporation
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-# pylint: disable=useless-object-inheritance, keyword-arg-before-vararg
+# pylint: disable=keyword-arg-before-vararg
 # disabled for python2 compatibility reasons
```

Comment:
I'm not sure about this check disable, is it still needed? This seems like it should be fixed probably but I haven't understood it in full yet.

Reply:
We stopped using pylint and switched to black, so probably we don't *need* any of these, but they may be useful depending on people's personal preferences?